src/components/ProjectList/ProjectListRow/index.jsx

```diff
@@ -11,8 +11,13 @@ const cleanName = str => {
 };
 
 const ProjectRow = ({ project }) => {
+  console.log('project list query', project);
```

Comment:
Remove this console log before merging. You should remove console logs before merging whenever possible.